compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinalgFunctionOutlining.cpp

```diff
@@ -97,7 +99,8 @@ class AMDAIELinalgFunctionOutliningPass
       : AMDAIELinalgFunctionOutliningBase(opts) {}
 
   void getDependentDialects(DialectRegistry &registry) const override {
-    registry.insert<linalg::LinalgDialect>();
+    registry.insert<linalg::LinalgDialect, scf::SCFDialect, LLVM::LLVMDialect,
+                    func::FuncDialect>();
```

Comment:
Do you need to add all these dialects dependency?

Reply:
I use all these dialects, so I think so. The linalg and func dialects are obvious, the scf dialect and llvm dialect are used now too: for creating scf.for ops that don't get unrolled (llvm attribute to prevent unrolling).